pkgs/top-level/python-packages.nix

```diff
@@ -1213,7 +1213,7 @@ in {
 
   rq = callPackage ../development/python-modules/rq { };
 
-  rx = callPackage ../development/python-modules/rx { };
+  rx = disabledIf (! pythonAtLeast "3.6") (callPackage ../development/python-modules/rx { });
```

Comment:
please move into expression

Reply:
Will do. I followed the convention set by other packages, such as docker-py, opencv, and meson, but I didn't see that many more packages use `disabled = ...;` instead.

Reply:
the disabledIf works anywhere, but the disabled attr is special to python packages. It's common to have interpreter specific disables, so it's more likely to be commonly used in python